### ISSUE TITLE
TMDL extension detects error on measure names containing non-ASCII characters, not wrapped in single quotes.

### ISSUE THREAD

**Issue by jol1** [microsoft/vscode-tmdl#2]
I've updated to the latest version of the TMDL VS Code extension and it's detecting the following error on a measure that computes the chi-squared test statistic over the currently filtered set of data.

![Image](https://github.com/user-attachments/assets/5a2c72bf-2f46-49bf-a712-3aa797963e16)

Because there are no spaces in the measure name, I believe that single quotes shouldn't by necessary. In any case, this is the way that Power BI Desktop serializes the TMDL file when I save as PBIP.

I suspect there is a bug with the way the TMDL VS Code extension handles non-ASCII characters.

**Comment by agusmba** [comment]
Yep, this is a bit bothersome since many of my Measures are one word with accents (i.e. `Café`), which show as errors in vscode although the code is generated by PBI Desktop on saving the report.

**Comment by jol1** [comment]
This issue appears to be resolved in the 1.1.0 version of the extension
